src/tfp_backend/Code_gen.ml

```diff
@@ -176,7 +180,18 @@ let pp_bijector ppf trans =
   let components =
     match trans with
     | Program.Identity -> []
-    | Lower lb -> [("Exp", []); ("AffineScalar", [lb])]
+    | Lower lb -> [("Exp", []); ("Shift", [lb])]
+    | Upper ub ->
+        [("Exp", []); ("Scale", [Expr.Helpers.float (-1.)]); ("Shift", [ub])]
+    | LowerUpper (lb, ub) ->
+        [ ("Sigmoid", [])
+        ; ("Scale", [Expr.Helpers.binop ub Operator.Minus lb])
```

Comment:
OCaml also has a "local open" syntax that can be helpful sometimes; not sure if it's worth it here yet you could make the top of this let be
```ocaml
let components = 
  let open Expr.Helpers in
  match trans with
...
```    
and that would give you access to `float` and `binop` here unqualified.